script/lint.js

```diff
@@ -99,20 +100,26 @@ const LINTERS = [{
   test: filename => filename.endsWith('.js') || filename.endsWith('.ts'),
   run: async (opts, filenames) => {
     const eslint = new ESLint({
-      cache: true,
+      // Do not use the lint cache on CI builds
+      cache: !process.env.CI,
+      cacheLocation: `node_modules/.eslintcache.${crypto.createHash('md5').update(fs.readFileSync(__filename)).digest('hex')}`,
       extensions: ['.js', '.ts'],
       fix: opts.fix
```

Comment:
So I guess the `fix` option only includes the fixes in the output but doesn't write it to disk? Seems like that was the main issue, and fixed by the `ESLint.outputFixes()` call below.